network-api/networkapi/wagtailpages/factory/buyersguide.py

```diff
@@ -243,6 +243,7 @@ class Meta:
     header = Faker("sentence")
     title = Faker("sentence")
     cta = SubFactory(PetitionFactory)
+    first_published_at = Faker("past_datetime", start_date="-30d", tzinfo=timezone.utc)
```

Comment:
Unrelated to this PNI index page work but I caught this bug which causes the article list on `/privacynotincluded/articles` not being sorted properly.

Reply:
Awesome